src/new_hope/week1/day4.clj

```diff
@@ -31,31 +31,36 @@
 (defn my-odd [x]
   (filter isOdd x))
 
-(facts "" (isOdd 2) => false)
-(facts "" (isOdd 3) => true)
-(facts "" (my-odd #{1 2 3 4 5}) => '(1 3 5))
-(facts "" (my-odd [4 2 1 6]) => '(1))
-(facts "" (my-odd [2 2 4 6]) => '())
-(facts "" (my-odd [1 1 1 3]) => '(1 1 1 3))
+(facts ""
+       (isOdd 2) => false
+       (isOdd 3) => true
+       (my-odd #{1 2 3 4 5}) => '(1 3 5)
+       (my-odd [4 2 1 6]) => '(1)
+       (my-odd [2 2 4 6]) => '()
+       (my-odd [1 1 1 3]) => '(1 1 1 3))
 
 ;Write	a	function	which	returns	true	if	the	given	sequence	is	a	palindrome.	Hint:	“racecar”
 ;does	not	equal	‘(\r	\a	\c	\e	\c	\a	\r)
-(defn palin
-  ([in rev]
-   (if (empty? in)
-     true
-     (if (= (first in) (first rev))
-       (recur (rest in) (rest rev))
-       false)
-     )
-    )
-  )
-(palin "stss" "stss")
-;(defn isPalindrome [st]
-;
-;  (palin (st) (reverse st))
-;  )
-;(false?	(__	'(1	2	3	4	5)))(true?	(__	"racecar"))
-;(true?	(__	[:foo	:bar	:foo]))
-;(true?	(__	'(1	1	3	3	1	1)))
-;(false?	(__	'(:a	:b	:c)))
\ No newline at end of file
+; cond
+
+(defn isPalindrome [st]
+  (defn palin [in rev]
```

Comment:
this definition is not local, it's put on the global level
if you need a local function, use let:
```clj
(let [palin (fn [in rev] ... )]
  (palin ... ))
```